jsonxx.cc

```diff
@@ -307,6 +307,7 @@ bool Object::parse(std::istream& input, Object& object) {
             delete v;
             break;
         }
+        JSONXX_ASSERT(object.value_map_.find(key) == object.value_map_.end());
```

Comment:
I think it is better to keep the old behaviour, but prevent the leak: check if the key already exists, and release the value if so.